.github/workflows/build-docker-images.yml

```
@@ -120,37 +138,70 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
           repository: ${{ secrets.DOCKERHUB_ORGANIZATION_NAME }}/${{ inputs.image-name }}
           short-description: ${{ inputs.image-description }}
+      - name: Export output digest
+        run: |
+          mkdir -p ${{ runner.temp }}/digests
+          digest="${{ steps.build-image.outputs.digest }}"
+          touch "${{ runner.temp }}/digests/${digest#sha256:}"
+          # generate suffix for digest file 
+          platform=${{ matrix.platform.arch }}
+          echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
```

https://docs.github.com/en/actions/reference/workflows-and-actions/variables#default-environment-variables `$RUNNER_TEMP` is available so you don't need to do anything for it.

You shouldn't use `${{...}}` in `run:` steps.
```suggestion
        env:
           DIGEST: ${{ steps.build-image.outputs.digest }}
        run: |
          mkdir -p $RUNNER_TEMP/digests
          digest="$DIGEST"
          touch "$RUNNER_TEMP/digests/${digest#sha256:}"
          # generate suffix for digest file 
          platform=${{ matrix.platform.arch }}
          echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
```

---

You are right! Fixed